src/main/java/net/fabricmc/loom/configuration/providers/LaunchProvider.java

```diff
@@ -197,7 +215,7 @@ public String getTargetConfig() {
 	}
 
 	public static class LaunchConfig {
-		private final Map<String, List<String>> values = new HashMap<>();
+		private final Map<String, List<String>> values = new LinkedHashMap<>();
```

Comment:
This one's a bit subtle but I couldn't think of any better solution. It all comes down to the fact that order of "--tweakClass" arguments matters. We add the default server/client forge tweakers to the "server"/"client" arguments right from the start, if the user adds their own "server" or "client" tweaker, all is fine, but if they add their own tweaker to the "common" arguments, then the actual order of the final arguments depends on HashMap iteration order because that's the order in which the three section ("common","client" and "server") are written to the dev-launch-injector file (common isn't merged into the other ones until the game actually starts; come to think of it, maybe we should just merge them at build time rather than runtime?).